lib/tasks/check-links/link_checker.rake

```diff
@@ -27,9 +27,17 @@ task "check-links": :environment do
 end
 
 namespace :"check-links" do
-  desc "Check links for a single local authority"
+  desc "Check links & update link status for a single local authority"
   task :local_authority, [:authority_slug] => :environment do |_, args|
     checker = LocalLinksManager::CheckLinks::LinkStatusRequester.new
     checker.check_authority_urls(args[:authority_slug])
   end
+
+  desc "Check links & update link status for all local authorities"
+  # This will run in the background as a queue of Sidekiq jobs.
+  # The jobs will take a long time to complete, even a few hours.
```

Comment:
If you include this comment in the description string, it'll be shown to users if they ask for help on the task which might be useful.